### ISSUE TITLE
Creating a DM with a user whose homeserver is down creates endless empty rooms 

### ISSUE THREAD

**Issue by frebib** [element-hq/element-x-android#2962]
I've submitted an in app rageshake.
The `_matrix/federation/v2/invite` request is failing with Connection Refused

**Comment by bmarty** [comment]
Can it be an issue server side @erikjohnston ?

The server is https://matrix.nerdhouse.io/, running Synapse `1.108.0`

**Comment by erikjohnston** [comment]
Yup, currently the server creates the room then tries to invite the user. If the client retries then it will create more rooms
